db/migrations/20161019_1230_remove_contracting_ori_field.rb

```diff
@@ -0,0 +1,37 @@
+# Migration to remove the 'ori' field for Incidents and
+# rename the GeneralInfo 'contracting_for_ori' field to 'ori'.
+class RemoveContractingOriField < DynamoDB::Migration::Unit
+  def update
+    logger = Logger.new(STDOUT)
+
+    logger.info "Running migration #{self.class.name}:"
+
+    incident_ids = Incident.all.map(&:id)
+    logger.info "Updating #{incident_ids.length} Incident records ..."
+    incident_ids.each do |id|
+      client.update_item(
+        table_name: "#{Dynamoid.config.namespace}_incidents",
+        key: { 'id' => id },
+        update_expression: "REMOVE ori"
```

Comment:
What would happen if it has no ori? Do we need a condition check here? I'm wondering about the case where a migration fails partway through and we have to re-run it (not totally unlikely)